articles/synapse_docker_registry.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Synapse Docker Registry
+layout: article
+excerpt: The basics of using the synapse docker registry
+---
+
+##Synapse Docker Registry
+Synapse now has its own docker registry which will allow you to have private docker repositories.  Please click [here](https://www.docker.com/products/overview) to learn how to use docker for mac, windows or linux. Below is a brief tutorial on how to use the synapse docker registry using the command line.
+
```

Comment:
1) The article will live on longer so the reference to "now" is unecessary.  
2) Don't use links to that say "click here" instead make the text that as the link.  For example To [learn more](...).
3) No need to focus on different systems.  The idea in this paragraph is to introduce the user to why they should care about this feature and what this article will teach them.  To achieve 1 you might need to start a little further back about what Docker is and why docker in Synapse is a good thing.  For 2 you should outline the kinds of operations you will go through.  Look at the Discussion and Teams articles for examples.